AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/hearing-details/hearing-details.component.spec.ts

```diff
@@ -150,23 +141,6 @@ describe('HearingDetailsComponent', () => {
         const result = component.getDefenceAdvocateByContactEmail('madeup@doesnotexist.com');
         expect(result).toBe('');
     });
-    describe('feature flag turn on and off ', () => {
-        it('should not show allocated to label if work allocation feature flag is off', async () => {
-            launchDarklyServiceSpy.getFlag.withArgs(FeatureFlags.vhoWorkAllocation).and.returnValue(of(false));
-            await component.ngOnInit();
-            fixture.detectChanges();
-            const divToHide = fixture.debugElement.query(By.css('#hearing-allocated-to'));
-            expect(divToHide).toBeFalsy();
-        });
-
-        it('should show allocated to label if work allocation feature flag is on', async () => {
```

Comment:
this test is still relevant as work allocation is always enabled. Just need to remove the line that sets the feature flag

Reply:
reinstated